src/flows/components/header/index.tsx

```diff
@@ -139,6 +199,81 @@ const FlowHeader: FC = () => {
       .catch(err => console.error('failed to delete share', err))
   }
 
+  const canvasOptions = {
+    backgroundColor,
+    onclone: cloneDoc => {
+      cloneDoc.getElementById(currentID).style.paddingLeft = '12px'
+      cloneDoc.getElementById('pipe-list-hidden-header').style.display = 'block'
```

Comment:
we should generalize this mechanism to using something like `data-download-show` and `data-download-hide` on the components, then using this part of the code to run `cloneDoc.querySelectorAll('data-download-hide').forEach(d => d.style.display = 'none')` so that we can decouple the header from the components that get rendered from the thing doing the rendering. for example, as we scale this out in it's current form, we'd have to keep adding element IDs here to cover all of the different elements we want to toggle and which way we want to toggle them. this creates an implicit relationship between this part of the code matching an id defined somewhere in a different file. when we change that file or delete it, we have to know to always come and check this file as well. by switching all of these to communicate our intended change during download mode through a common interface, we can decouple the definition and mutation parts and scale this to whatever complexity you want.